docs/hooks/useAlert.md

```diff
@@ -1,18 +1,26 @@
 # `useAlert`
 
-`useAlert(message, options?) → { show }`
+`useAlert(message, options?) → { show, hide }`
 
 ## Hook arguments
 
-| Name      | Type                   | Description                                                                                                                                                                                                                                                              |
-| --------- | ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `message` | `string` or `Function` | The message to display                                                                                                                                                                                                                                                   |
-| `options` | `object` or `Function` | A configuration object that matches [the props of the `AlertBar`](https://ui.dhis2.nu/demo/?path=/docs/feedback-alerts-alert-bar--default) in `@dhis2/ui` (alternate: [minimal view](https://ui.dhis2.nu/#/api?id=coresrcalertbaralertbarproptypes-object) of the props) |
+| Name      | Type                   | Description                                                                                                 |
+| --------- | ---------------------- | ----------------------------------------------------------------------------------------------------------- |
+| `message` | `string` or `Function` | The message to display                                                                                      |
+| `options` | `object` or `Function` | A configuration object for the alert. See [note](#note) below for usage instructions in DHIS2 platform-apps |
+
+## Usage in DHIS2 platform-apps
+
+The DHIS2 app-shell comes with the alerts provider and a component to show `AlertBar`s in an `AlertStack` (`@dhis2/ui` components), so in a typical DHIS2 platform-app only the `useAlert` hook is used and it can be imported from `@dhis2/app-runtime`.
```

Comment:
```suggestion
The DHIS2 app-shell comes with the alerts provider and a component to show `AlertBar`s in an `AlertStack` (a `@dhis2/ui` components), so in a typical DHIS2 platform app only the `useAlert` hook is used and it can be imported from `@dhis2/app-runtime`.
```